### PR TITLE
Fixing the issue of multiple output in the duplicates dialog 

### PR DESCRIPTION
**Pull request by lilyclements**
@africanmathsinitiative/developers this is ready for review

I noticed a very minor bug in the Duplicates dialog where if you clicked reset and clicked "OK", then the `summary` code was run multiple times. This PR fixes this so that this should not be an issue.

To test:
* Load data
* Open dialog (Prepare > Check Data > Duplicates)
* Click "Reset"
* Click "OK"
* The `summary` output should now only be printed once

### CODE REVIEW

**Review by dannyparsons — APPROVED** [review]
Code looks correct. A reviewer should also check functionality.

File: instat/dlgDuplicates.vb [hunk 1]
```diff
@@ -137,8 +137,6 @@ Public Class dlgDuplicates
         clsStreakFunction = New RFunction
         clsSubsetCol = New RFunction
         clsDupCountIndex = New RFunction
-        clsGetColumnsFunction = New RFunction
-        clsSummaryFunction = New RFunction
```
Comment (shadrackkibet) [inline instat/dlgDuplicates.vb:141]: Do you have `AddToAfterCodes/AddToBeforeCodes` somewhere? I think the correct way to fix this is by clearing codes i.e `   ucrBase.clsRsyntax.ClearCodes()` in `SetDefaults()`.

Comment (dannyparsons) [reply instat/dlgDuplicates.vb:141]: Thanks Shadrack, I read this differently at first, I thought this was defining the RFunctions again. Yes, these lines should stay here and there should be a clearing of codes as Shadrack mentioned, which should solve the issue.

Comment (dannyparsons) [reply instat/dlgDuplicates.vb:141]: An alternative would be to do `clsSummaryFunction.Clear()`. But I think I prefer explicitly clearing the codes as Shadrack mentioned.

Comment (lilyclements) [reply instat/dlgDuplicates.vb]: Thanks for this. I have sorted it now and so this is ready for a re-review

**Review by rdstern — APPROVED** [review]

**Review by shadrackkibet — APPROVED** [review]
Looks good. I therefore approve.

**Pull request merged**